### PR TITLE
Add conda recipe for json-rpc, using meta.yaml

### PR DESCRIPTION
**Pull request by dharhas**
tweaked from 'conda skeleton pypi json-rpc'

### CODE REVIEW

**Comment by dharhas** [comment]
@ocefpaf the circleci failure seems to be related to a different package 'cyavro' and not this pull request.

**Comment by ocefpaf** [comment]
@dharhas can you update and rebase you fork? There may be a few recipes remaining. but you can delete them for now until we "fix" Travis-CI.

**Comment by dharhas** [comment]
sure

**Comment by ocefpaf** [comment]
Nice. Now you are "contaminated" only by `redis-py` and `singledispatch`. Feel free to delete them from you PR so the CIs work only on your recipe.

Note: nothing that is happening here is your fault @dharhas. It is a consequence of our use of `Travis-CI` and some outages and issue we are having with it lately.

PS: See you in 10 day in Austin?

**Comment by dharhas** [comment]
Done. Yup, see you at SciPy.

File: recipes/json-rpc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
```
Comment (ocefpaf) [inline recipes/json-rpc/meta.yaml]: @dharhas please remove the remaining bolierplate comment from conda-skeleton.

**Comment by jakirkham** [comment]
Well, this one is stuck.

**Comment by jakirkham** [comment]
@pavlov99, just letting you know we have added `json-rpc` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge json-rpc`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/json-rpc-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `json-rpc`. 😄

**Pull request merged**